packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -190,6 +194,19 @@ export const AvailableTimes = ({
 }: AvailableTimesProps) => {
   const { t } = useLocale();
 
+  if (slots[0] && slots[0].away) {
+    const { toUser, fromUser, reason, emoji } = slots[0];
```

Comment:
Receiving new props in slots and returning UI early for displaying new UI that allows to switch to different profile.